### PR TITLE
chore: bump deps

### PR DESCRIPTION
**Pull request by frezbo**
Bump dependencies. Ref:

- #225
- #226

Signed-off-by: Noel Georgi <git@frezbo.dev>

### CODE REVIEW

File: bash/pkg.yaml [hunk 1]
```diff
@@ -1,15 +1,87 @@
...
+  - stage: patch
+    runtime: true
 steps:
   - sources:
       - url: https://ftp.gnu.org/gnu/bash/bash-{{ .bash_version }}.tar.gz
         destination: bash.tar.gz
         sha256: "{{ .bash_sha256 }}"
         sha512: "{{ .bash_sha512 }}"
+      # ref https://git.alpinelinux.org/aports/tree/main/bash/APKBUILD?h=3.16-stable
+      - url: https://ftp.gnu.org/gnu/bash/bash-5.1-patches/bash51-001
```
Comment (smira) [inline bash/pkg.yaml:13]: just a small set of patches :eyeglasses:

**Review by smira — APPROVED** [review]

File: bash/pkg.yaml (hunk 1 above)
Comment (frezbo) [reply bash/pkg.yaml:13]: yes, bash doesn't release newer versions, instead patches, so this is something that even if renovate updates, there could be patches released which we'll have to pick upstream.

**Comment by frezbo** [comment]
This PR also fixes the reproducibility

**Pull request merged**